concepts.md

```diff
@@ -193,13 +195,12 @@ When received by a *device*, they are stored in the [Local Encrypted Storage].
 
 ### Verification key
 
-The [Verification Key] is used to register new devices in a secure manner. See the [procotol documentation](protocol.md) for more details.
+The [Verification Key] is used to register new devices in a secure manner. See the [protocol documentation](protocol.md) for more details.
 
 ### Secret Provisional Identity
 
 A [Secret Provisional Identity] represents the identity of a user that is not yet registered on Tanker. It is split into two halves that are stored on the *application server* and on *Tanker server*s.
-A provisional identity is attached to some authentication methods. For the moment only email is supported.
-Each half contains the name of the authentication method, a value (in case of email, the value is the email address), and encryption and signature key pairs.
+Each half contains the name of the authentication method, a value (in case of email, the value is the email address), an encryption and a signature key pairs.
```

Comment:
`an encryption key pair, and a signature key pair` or the previous sentence which was correct too

Reply:
Not sure what difference it makes...